packages/backend/src/ee/database/migrations/20250713101551_add_query_history_account.ts

```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+
+const QUERY_HISTORY_TABLE = 'query_history';
+const CREATED_BY_ACCOUNT_COLUMN = 'created_by_account';
+const RESULTS_EXPIRES_AT_COLUMN = 'results_expires_at';
+
+export async function up(knex: Knex): Promise<void> {
+    // 1. Add a new nullable "createdByAccount" column of type string
+    await knex.schema.alterTable(QUERY_HISTORY_TABLE, (table) => {
+        table.string(CREATED_BY_ACCOUNT_COLUMN).nullable();
+    });
+
+    // 2. Add indexes on the new column and the expired column
+    await knex.schema.alterTable(QUERY_HISTORY_TABLE, (table) => {
+        table.index([CREATED_BY_ACCOUNT_COLUMN]);
+        table.index([RESULTS_EXPIRES_AT_COLUMN]);
```

Comment:
indexing the existing expires_at column is the only expensive migration. if you omit this you can make these changes and move on independently of scheduling in the bigger migration for expires at and handling deletes